### PR TITLE
docs: update readme to reference docs site

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,279 +26,19 @@ Join us to help define the direction and implementation of this project!
...
-Run unit tests locally with `make test`.
-
-### End-to-end Tests
-
-End-to-end tests automatically runs on Prow when a PR is submitted. If you want to run using a local or remote Kubernetes cluster, make sure to have `kubectl`, `helm` and `bats` set up in your local environment and then run `make e2e-azure` or `make e2e-vault` with custom images.
-
-Job config for test jobs run for each PR in prow can be found [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-config.yaml)
-
-## Known Limitations
+![Secrets Store CSI Driver Demo](/docs/book/src/images/demo.gif "Secrets Store CSI Driver Azure Key Vault Provider Demo")
```
Comment (ritazh) [inline README.md:33]: where is this file: demo.gif

Comment (aramase) [reply README.md]: Updated with the correct location

**Pull request merged**